src/clr-angular/data/datagrid/datagrid-cell.spec.ts

```diff
@@ -43,6 +43,10 @@ export default function(): void {
       context.detectChanges();
       expect(context.clarityElement.classList.contains('datagrid-signpost-trigger')).toBeTruthy();
     });
+
+    it('adds a11y roles to the rolumn', function() {
```

Comment:
rolumn => cell


:joy: at "rolumn" by the way.

Reply:
cost of copy-paste.. T_T thanks for catching!